core/src/main/scala/com/karumi/shot/base64/Base64Encoder.scala

```diff
@@ -0,0 +1,22 @@
+package com.karumi.shot.base64
+
+import java.io.{ByteArrayOutputStream, File}
+import java.util.Base64
+import javax.imageio.ImageIO
+
+import org.apache.commons.io.Charsets
+
+object Base64Encoder {
+
+  def base64FromFile(filePath: String): String = {
```

Comment:
The generation could fail in scenarios where the file used as source is not generated properly. I'd group this inside a Try catch block and change the return value to be Option[String]. Closing the output stream inside a finally block should be also interesting.

Reply:
You are right! I will take a look, just for curiosity, in which scenarios might fail?

Reply:
Ahhh okey, I have just seen the different errors that could happen in the `showErrors`